src/features/components/templates/banners/blade-shape-banner/index.tsx

```diff
@@ -78,64 +78,97 @@ const BladeShapeBanner = ({ data }: { data: BannerType }) => {
+                                    </Typography.Paragraph>
+                                    <Typography.Heading
+                                        size={'xxs'}
                                         textcolor={'white'}
-                                        size={'large'}
-                                        weight={'bold'}
+                                        align={'left'}
                                     >
-                                        {item.data.text}
-                                    </Typography.Paragraph>
-                                </Link>
-                            </Flex.Box>
-                        ))}
-                    </Flex.Box>
-                </Flex.Box>
-            </Flex.Box>
+                                        <Localize
+                                            translate_text={data.traders_hub_info.heading_two}
+                                        />
+                                    </Typography.Heading>
+                                </FlexBox.Box>
+                            </FlexBox.Box>
+                        </Link>
+                    )}
+                    <FlexBox.Box
+                        className={styles.qr_box}
+                        direction={'col'}
+                        align={'start'}
+                        padding={'8x'}
+                        md={{ padding: '12x' }}
+                    >
+                        <FlexBox.Box direction={'row'} align={'center'} md={{ mb: '3x' }}>
+                            <Image
+                                src={data.scan_box.icon}
+                                alt={'_t_qr code_t_'}
+                                width={64}
+                                height={64}
+                            />
+                            <FlexBox.Box direction={'col'} ml={'8x'} md={{ ml: '12x' }}>
+                                <Typography.Paragraph textcolor={'white'}>
+                                    <Localize translate_text={data.scan_box.heading_one} />
+                                </Typography.Paragraph>
+                                <Typography.Heading size={'xxs'} textcolor={'white'}>
+                                    <Localize translate_text={data.scan_box.heading_two} />
+                                </Typography.Heading>
+                            </FlexBox.Box>
+                        </FlexBox.Box>
+                        <FlexBox.Box wrap={'wrap'}>
+                            {data.os_apps.map((item) => (
+                                <FlexBox.Box key={item.id} pt={'12x'} basis={'6-12'}>
+                                    <Link pl={'3x'} url={item.data.url} no_hover>
+                                        <FlexBox.Box align={'center'}>
+                                            <Image
+                                                src={item.data.icon}
+                                                alt={item.data.text}
```

Comment:
alt is not localized here?

Reply:
Once we'll move all images to bff and from this place we can't localize it
so, have no sense to spent time on it